tests/utils/test_notifications.py

```diff
@@ -496,6 +544,25 @@ def test_notification_validation_on_object(
         mlrun.model.Notification(**notification_kwargs)
 
 
+def test_notification_validation_defaults(monkeypatch):
+    notification = mlrun.model.Notification()
+    notification_fields = {
+        "kind": mlrun.common.schemas.notification.NotificationKind.slack,
+        "message": "",
+        "severity": mlrun.common.schemas.notification.NotificationSeverity.INFO,
+        "when": ["completed"],
+        "condition": "",
+        "name": "",
+    }
+
+    for field, expected_value in notification_fields.items():
+        value = getattr(notification, field)
+        assert value is not None, f"{field} field in the notification object is None"
+        assert (
+            value == expected_value
+        ), f"{field} field value is {value}, expected {expected_value}"
```

Comment:
The first assert is a bit redundant when you have the second one

Reply:
what if the field is not None but is not the expected?

Reply:
Your test isn't checking that scenario. It's testing an empty notification